Popular_Models_Guide/Llama2/trtllm_guide.md

```diff
@@ -269,17 +269,19 @@ and [perf_best_practices](https://github.com/NVIDIA/TensorRT-LLM/blob/main/docs/
 # preprocessing
 TOKENIZER_DIR=/Llama-2-7b-hf/
 TOKENIZER_TYPE=auto
+ENGINE_DIR=/engines
 DECOUPLED_MODE=false
 MODEL_FOLDER=/opt/tritonserver/inflight_batcher_llm
 MAX_BATCH_SIZE=4
 INSTANCE_COUNT=1
 MAX_QUEUE_DELAY_MS=10000
+TRITON_BACKEND=tensorrtllm
```

Comment:
Let's add a note above that there are 2 options available, `tensorrtllm` and `python`. Also, let's note that python backend will deploy `model.py`